pandas/core/indexes/interval.py

```diff
@@ -788,7 +788,7 @@ def _find_non_overlapping_monotonic_bounds(self, key):
         return start, stop
 
     def get_loc(
-        self, key: Any, method: Optional[str] = None
+        self, key: Any, method: Optional[str] = None, tolerance=None
```

Comment:
Is this required because of inheritance? Not used in function body AFAICT so somewhat odd

Reply:
yes. subclasses should have the same or additional arguments. not less.